tests/test_cookie.py

```diff
@@ -97,14 +99,18 @@ def test_check_conda_environment_creation_and_run_all_checks(cookies):
             ("git", "checkout", "-b", "test"), cwd=result.project_path, check=True
         )
 
+        conda_exe = os.environ["CONDA_EXE"]
+
         # Check linting, but not on the first try since formatters fix stuff.
         subprocess.run(
-            ("bash", "-l", "-c", "conda run -n __test__ pre-commit run --all-files"),
+            (conda_exe, "run", "-n", "__test__", "pre-commit", "run", "--all-files"),
```

Comment:
Is the `is_ci` flag necessary after this change any more?

Reply:
The `is_ci` flag is used by `paper/task_paper.py` to switch between `infer_latex_dependencies` and the ones that are manually set.

Note: Instead of manually setting it in the tests we could check if we are in a CI environment.

Reply:
Ah, I remember, sorry. Thought it was only about my local environment problem with the last test...

> Note: Instead of manually setting it in the tests we could check if we are in a CI environment.

If that is easy to pick up in cookiecutter (I guess some environmental variable could be checked?), that would be the better option.

Reply:
I spent a few minutes reading the cookiecutter documentation but couldn't figure out how to do it. In principle, we just need to create a cookiecutter variable dependent on the environment state. I think this is definitely possible and the preferred option but I cannot judge how long this would take, could be anywhere between minutes and days.

But before implementing this we could still set it automatically in the tests. Then one version is tested remotely and the `infer_latex_dependencies` version is tested locally.

Reply:
Cool. Do you know out of the top of your head which environmental variable I need to check for whether a run is taking place within a Github Actions environment?

Reply:
I think its just called `CI`. I remember they had a list [here](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables).